test/e2e/e2e_leaderelection_test.go

```diff
@@ -33,9 +33,19 @@ import (
 	clientset "k8s.io/client-go/kubernetes"
 	utilptr "k8s.io/utils/ptr"
 	"sigs.k8s.io/descheduler/cmd/descheduler/app/options"
+	"sigs.k8s.io/descheduler/pkg/api"
 	"sigs.k8s.io/descheduler/pkg/descheduler"
 )
 
+// Should use something like "sigs.k8s.io/descheduler/pkg/framework/plugins/removepodshavingtoomanyrestarts"
```

Comment:
yes, in this case it would be `"sigs.k8s.io/descheduler/pkg/framework/plugins/podlifetime"`.